### PR TITLE
Print IDs

### LINKED ISSUE #422: Print VMID, Blockchain ID, and SubnetID for subnet
`subnet describe` and `subnet list` should print out the VMID for the config.

### PR DESCRIPTION
**Pull request by holisticode**
Closes #422 

Refactored `list` into 2 separate commands, `list "general"` (no option, default) and `list --deployed` *(shows deploy info)

Probably requires small adjustments to docs

### CODE REVIEW

**Review by dasconnor — COMMENTED** [review]
couple code comments, I also want to run this locally and see what the output looks like

File: cmd/subnetcmd/list.go [hunk 1]
```diff
@@ -38,21 +44,101 @@ func (c subnetMatrix) Swap(i, j int) { c[i], c[j] = c[j], c[i] }
 func (c subnetMatrix) Less(i, j int) bool { return strings.Compare(c[i][0], c[j][0]) == -1 }
 
 func listSubnets(cmd *cobra.Command, args []string) error {
-	header := []string{"subnet", "chain", "chain ID", "type", "from repo", "", "deployed", ""}
+	if deployed {
+		return listDeployInfo(cmd, args)
+	}
+	header := []string{"subnet", "chain", "chain ID", "vm ID", "type", "from repo"}
```
[the remark this answers is not in the document]
Comment (holisticode) [reply cmd/subnetcmd/list.go:50]: The capitalization has no effect, this is going to be printed all uppercase by the tablewriter.

File: cmd/subnetcmd/list.go [hunk 2]
```diff
@@ -74,70 +160,80 @@ func listSubnets(cmd *cobra.Command, args []string) error {
...
-		}
-		for _, f := range files {
-			if f.Name() == constants.SidecarFileName {
-				carName := s.Name()
-				// read in sidecar file
-				sc, err := app.LoadSidecar(carName)
-				if err != nil {
-					return err
-				}
+	fujiK := models.Fuji.String()
```
[the remark this answers is not in the document]
Comment (holisticode) [reply cmd/subnetcmd/list.go:168]: Yes, `key`. I will rename, although usually for variables with very local scope, it is ok to use abbreviations which live for short time https://dave.cheney.net/practical-go/presentations/qcon-china.html#_identifiers

[the remark this answers is not in the document]
Comment (dasconnor) [reply cmd/subnetcmd/list.go]: I'm fine with short abbreviations, the abbreviation just wasn't clear to me, I thought it was confusing.

**Review by dasconnor — APPROVED** [review]

**Pull request merged**